src/GitHub.VisualStudio.UI/Views/GitHubPane/PullRequestDetailView.xaml

```diff
@@ -248,7 +248,8 @@
                                       HeaderText="Checks"
                                       IsExpanded="True"
                                       Margin="0 8 0 0"
-                                      ghfvs:ScrollingVerticalStackPanel.IsFixed="true">
+                                      ghfvs:ScrollingVerticalStackPanel.IsFixed="true"
+                                      Visibility="{Binding HasChecks, Converter={ghfvs:BooleanToVisibilityConverter}}">
```

Comment:
Rather than adding a `HasChecks` property you can use the `CountToVisibilityConverter`.